examples/LAPACK/foo.cpp

```diff
@@ -0,0 +1,14 @@
+#include <iostream> // std::cout
+#include <cblas.h>
```

Comment:
I think we are conflicting with OpenBLAS package here:
* https://github.com/hunter-packages/OpenBLAS/blob/hunter/cblas.h

Do you know how they are related? Is it two implementation of the same standard?